docs/client-connectors/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Client Connectors - Beta",
```

Comment:
Are client connectors a beta feature? I assumed they are, because connectors are.

Reply:
@diana-macrometa - I believe so. But let's check with @elof / @dustinlarimer / @rahulprabhune-macrometa first.

Reply:
@elof, @dustinlarimer - Please answer this question.

Reply:
My opinion is that, if regular connectors are beta, than client connectors will also be considered beta. @diana-macrometa @grainier-macrometa @dustinlarimer